LICENSE

```diff
@@ -187,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2017 The Kubernetes Authors
```

Comment:
We can do what is done here: https://github.com/kubernetes/kubernetes/blob/master/LICENSE#L190